packages/wrangler/src/pages/dev.tsx

```diff
@@ -228,6 +230,12 @@ export const Handler = async ({
 		if (!existsSync(scriptPath)) {
 			logger.log("No functions. Shimming...");
 			scriptPath = resolve(__dirname, "../templates/pages-shim.ts");
+		} else {
+			workerJsBuild({
+				entryPoints: [scriptPath],
+				write: false,
+				plugins: [blockWorkerJsImports],
+			}).catch((err) => console.error("Failed to build _worker.js:", err));
```

Comment:
```suggestion
			try {
				workerJsBuild({
					entryPoints: [scriptPath],
					write: false,
					plugins: [blockWorkerJsImports],
				})
			} catch {}
```

The `console.error` seems redundant if we're already logging the error in the plugin.